homeassistant/components/anova/sensor.py

```diff
@@ -49,11 +52,15 @@ class AnovaSensorEntityDescription(
     AnovaSensorEntityDescription(
         key="state",
         translation_key="state",
+        device_class=SensorDeviceClass.ENUM,
+        options=[state.name for state in AnovaState],
         value_fn=lambda data: data.state,
     ),
     AnovaSensorEntityDescription(
         key="mode",
         translation_key="mode",
+        device_class=SensorDeviceClass.ENUM,
+        options=[mode.name for mode in AnovaMode],
         value_fn=lambda data: data.mode,
```

Comment:
What were the possible states before? Is this a breaking change?

Reply:
So the underlying value is going to be different. As before the value returned from the api would be "Low water" for example. Whereas now it is "low_water" and it is translated to "Low water"

Reply:
Then, it is a breaking change, and we need to document it as one. Please fill out the breaking change section

Reply:
Updated the breaking change section